Documentation/configuration/metrics.rst

```diff
@@ -41,7 +41,20 @@ Endpoint
 Datapath
 --------
 
-* ``datapath_errors_total``: Total number of errors occurred in datapath management, labeled by area, name and address family.
+* ``datapath_errors_total``: Total number of errors occurred in datapath
+  management, labeled by area, name and address family.
+* ``datapath_conntrack_gc_deleted``: Number of deleted entries in the
```

Comment:
Given this is a counter, it should have the _total suffix

Reply:
This is a old comment, now it's a gauge.

Reply:
It should still have units suffix, for example "_count"